combat-trainer.lic

```diff
@@ -352,6 +355,12 @@ class LootProcess
     end
   end
 
+  def should_perform_ritual(game_state)
+    unless game_state.prepare_consume
+      (DRSkill.getxp('Thanatology') <= 30 || (@necro_train_first_aid && DRSkill.getxp('First Aid') <= 30)) || !@skin
+    end
```

Comment:
Agreed, the logic is a bit of a mess. My brain was hurting there for awhile.

So the reason for the change is that it wasn't training First Aid if Thanatology was locked.  So my goal is to have it perform the ritual if EITHER are not locked.  I'd also want to leave the training threshhold at 31 since you can get 2.x mindstates from rituals.